OpenRA.Mods.Common/Warheads/CreateEffectWarhead.cs

```diff
@@ -30,6 +30,9 @@ public class CreateEffectWarhead : Warhead
 		[Desc("Remap explosion effect to player color, if art supports it.")]
 		public readonly bool UsePlayerPalette = false;
 
+		[Desc("Search radius around impact for 'direct hit' check.")]
+		public readonly WDist TargetSearchRadius = new WDist(3072);
```

Comment:
IIRC we have a lint check for testing these radii.  You'll need to add this there.